src/js/utils/useGlobalNav.js

```diff
@@ -1,16 +1,30 @@
 import { useEffect, useState } from 'react';
 import { load } from 'js-yaml';
-import { getNavFromConfig } from '../nav/globalNav';
 import sourceOfTruth from '../nav/sourceOfTruth';
 
-const appIds = ['insights', 'openshift', 'cost-management', 'subscriptions', 'ansible', 'settings'];
-
-const extraApps = [
+// TODO: add App services 26.4. for release
+const allowedApps = [
+  { id: 'openshift', title: 'Openshift', routes: [{ id: '', title: 'Clusters' }, { id: 'subscriptions' }, { id: 'cost-management' }] },
+  {
+    id: 'insights',
+    title: 'Red Hat Enterprise Linux',
+    routes: [
+      { id: 'dashboard' },
+      { id: 'advisor' },
+      { id: 'drift' },
+      { id: 'inventory' },
+      { id: 'vulnerability' },
+      { id: 'compliance' },
+      { id: 'policies' },
+      { id: 'patch' },
+      { id: 'subscriptions' },
+      { id: 'remediations' },
+    ],
+  },
+  { id: 'ansible', routes: [{ id: 'automation-hub' }, { id: 'catalog' }, { id: 'automation-analytics' }] },
   {
-    title: 'Insights for SAP',
-    id: 'SAP',
-    routes: [{ id: 'sap-dashboard', title: 'Dashboard' }],
-    isEntitled: async () => await insights.chrome.auth.getUser().entitlements?.insights?.is_entitled,
+    id: 'settings',
+    routes: [{ id: 'my-user-access' }, { id: 'rbac' }, { id: 'sources' }, { id: 'integrations' }, { id: 'notifications' }, { id: 'applications' }],
```

Comment:
As per the latest CRC changes, I think we should also exclude `notifications` from the list.

cc @karelhala